config/samples/layout/base/glance_v1beta1_glance.yaml

```diff
@@ -11,5 +11,3 @@ spec:
   databaseInstance: openstack
   databaseAccount: glance
   secret: osp-secret
-  storageClass: local-storage
-  storageRequest: 10G
```

Comment:
Should this also have a `storage` replacement like the changes to the other samples?

Reply:
I removed it because the other files (under layout/) replace this part w/ kustomize, but I think it's a good idea to keep it in case we need it for future layouts.

Reply:
I will update it in the next PS

Reply:
Done